<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>1230</width>
      <height>420</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">CST</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descricao</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sit. Trib.</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Op. Simples</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>450</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(132, 168, 163);
color: rgb(33, 25, 255);
font: 87 16pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>CST Cadastradas</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1230</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>INCLUSAO - ALTERACAO</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>30</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>CST</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="mcst">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>70</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>30</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Opt. Simples</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>30</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Sit. Tributaria</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>820</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Descricao</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_salvar">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>30</y>
       <width>90</width>
       <height>31</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;file:///C:/Users/mebbc/OneDrive/Imagens/salvar.qrc.png&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string> Salvar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>SALVAR.png</normaloff>SALVAR.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_cancelar">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>70</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string> Cancelar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>SAIR.png</normaloff>SAIR.png</iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="mdescri">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>820</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>120</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>70</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>70</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bt_sair">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>190</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> SAIR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>SAIR.png</normaloff>SAIR.png</iconset>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_alteracao">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Alteracao</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_inclusao">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>190</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> Inclusao</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_exclusao">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Exclusao</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>rb_alteracao</tabstop>
  <tabstop>rb_exclusao</tabstop>
  <tabstop>bt_inclusao</tabstop>
  <tabstop>bt_sair</tabstop>
  <tabstop>mcst</tabstop>
  <tabstop>mdescri</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>bt_salvar</tabstop>
  <tabstop>bt_cancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
